### PR TITLE
Rough outline for the Seminar Series

### PR DESCRIPTION
**Pull request by samumantha**
Title is preliminary and will be updated to the poll winner on zulip before merging.

Adjusted some of the unconference text to fit the series. Please read carefully if it fits this series as well :)

@PatricHolmvall : Anything from your original text that should be mentioned here, maybe?

All text is open for improvement/fully rewriting, this is just a starting point.

Do we need the section 'Nordic-RSE provides' from the unconference page here as well? Including Code of Conduct?

### CODE REVIEW

**Review by bast — APPROVED** [review]
I think it's a great starting point and we can refine later.

Minor note for me to check what the "description" does at all and whether we need the "subtitle". I introduced this as workaround on the other page but maybe we could use the description instead.

**Comment by PatricHolmvall** [comment]
Thank you for your work, great starting point!

**Review by PatricHolmvall — APPROVED** [review]

File: content/events/seminar-series/_index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Are you developing software or tools that are driven by research/engineering in either academia or industry? 
+
+Need to network, share knowledge and experiences with your peers? 
+
+Then you came to the right page: Nordic-RSE invites everyone interested to participate in Nordis-RSE seminars:
+
+The series will start out as a **monthly event in August 2021**. The time for each seminar will be set by the speaker. We are inviting everyone to suggest topics and/or speakers for this series (link coming soon).
+
+To kick off the event some of our members will introduce you to their work at Nordic universities.
+
```
Comment (rkdarst) [inline content/events/seminar-series/_index.md:18]:
```suggestion

## Format

As an example, events can be 60 (40+20 discussion) or 30 (15+15 discussion) minutes.  Events could be talks, demos, discussion, debates, and so on.  Events are hosted with HackMD for public anynchronous discussion.  We will provide mentoring for speakers.

```

**Review by rkdarst — APPROVED** [review]
This looks good to me too, we just need the name.  Several minor suggestions I will accept myself, one for someone else to look at.

File: content/events/seminar-series/_index.md (hunk 1 above)
Comment (PatricHolmvall) [reply content/events/seminar-series/_index.md:18]: Good idea to add sub-headers by the way. We could maybe add some more, and use other ways to delimit/highlight the text.

**Comment by samumantha** [comment]
Great suggestions! Thanks! I can work on it a bit more during next few days, so if you have more suggestions or something missing, let me know :)

File: content/events/seminar-series/_index.md (hunk 1 above)
Comment (samumantha) [reply content/events/seminar-series/_index.md:18]: Mini symposium could be great as well. Maybe we can leave the number of talks per seminar open and see how it goes and adjust accordingly?
Yes more subheaders and other highlighters could be added, will take a look at it

**Comment by PatricHolmvall** [comment]
> Great suggestions! Thanks! I can work on it a bit more during next few days, so if you have more suggestions or something missing, let me know :)

Thanks for all the great work. We can also draw a line at some point and leave things for future PRs, so you don't end up working on this forever. I'm very happy with this as a start.

**Pull request merged**